src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -4321,6 +4338,36 @@ int wm_vuldet_generate_win_cpe(agent_software *agent) {
     return 0;
 }
 
+int wm_vuldet_discard_kernel_package(agent_software *agent, char *name, char *version, char *arch) {
+    char *kernel_uname = NULL;
+
+    if (agent->dist == FEED_REDHAT || agent->dist == FEED_CENTOS) {
+        if (strcmp(name, "kernel")) {
```

Comment:
What about `kernel-rt` package?

Reply:
Didn't know about this one, I'll check it out! Thanks!